source/switching_to_live/index.html.md.erb

```diff
@@ -1,37 +1,93 @@
 ---
-title: Switching to live
+title: Go live
 weight: 100
 ---
 
-# Switching to live
+# Go live
 
-Please read this information carefully.
+## Before you start
 
-Please read the guidance on what to do [before you switch to live](/switching_to_live/before_you_switch_to_live/#before-you-switch-to-live) first.
+You must:
 
-Follow these steps to switch your account from ‘test’ to ‘live’. You should
-only do this when you’ve finished [testing with your test
-account](/testing_govuk_pay/), and you’re confident that
-[you can make payments successfully](/making_payments/#making-payments).
+- have successfully finished [testing your integration using your test account](/testing_govuk_pay/#testing-your-integration)
+- have a secure domain for your service, because you must use HTTPS for the `return_url` when you [create a payment](/making_payments/#creating-a-payment) in a live service
+- sign up for updates on the [GOV.UK Pay status page](https://payments.statuspage.io), so you get a notification when there's an incident
+- know how to [contact us](/support_contact_and_more_information/#contact-us) to report an incident
```

Comment:
Hmm good question. We tend to use 'incident' to mean an incident at our end (eg on https://docs.payments.service.gov.uk/support_contact_and_more_information/#incident-severity). @alexbishop1 Do you think this bullet should be more like 'know how to contact us if you need help'?

Reply:
Tracy is working on the new support page: https://docs.google.com/document/d/1z3mkc2xYIID0Dtq6_e1gGGzKt3xEZUMfMge1XA7Bu2E/edit#heading=h.opn41hy3qlhu so I think this text should be fine. We make it clearer how to contact us in case of an incident.

Reply:
I think the “incident” wording is supposed to imply knowing how to contact us in an emergency. We don’t publish the emergency email address anywhere (because emailing it will cause PagerDuty to wake up whoever’s on call) so it’s not something a service can quickly find out when they need it (unlike, say, the regular support address).

Reply:
I think the issue was that the support page it linked to doesn't really actually say how to contact us in an incident (since we can't share the email address, it's a bit more complicated), but that will shortly be fixed.

And yes, since the statement is - "what do you need to know before going live", then then it is fine to say incident. Good to go!